Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -111,14 +111,13 @@ class PlaceDetailsCardViewController: UIViewController {
     private func setLocation(location: CLLocation?) {
         if let lastTravelTimes = place.lastTravelTime {
             self.cardView.updateTravelTimesUI(travelTimes: lastTravelTimes)
-            return
```

Comment:
(continuing on the conversation from [the changeset that added this](https://github.com/mozilla-mobile/prox/pull/109#discussion_r86759933)) Let's add a comment explaining why we don't return here.